pom.xml

```diff
@@ -107,7 +107,10 @@
     <log4j.version>1.2.17</log4j.version>
     <disruptor.version>3.3.6</disruptor.version>
     <slf4j.version>1.7.25</slf4j.version>
-    <protobuf-java.version>2.5.0</protobuf-java.version>
+    <!-- com.google repo will be used except on Aarch64 platform. -->
+    <protobuf.group>com.google.protobuf</protobuf.group>
+    <protobuf-java.version>2.6.1</protobuf-java.version>
```

Comment:
I've updated to 2.6.1 to keep the x86_64 and aarch64 versions in sync.

Reply:
Reverted to 2.5.0 for both x86_64 and aarch64!
The OpenLabTesting guys released 2.5.0.2 which brings statically linked Protoc.
Here is its build script: https://gist.github.com/liusheng/64aee1b27de037f8b9ccf1873b82c413